MultiVI_tutorial.ipynb

```diff
@@ -0,0 +1,741 @@
+{
```

Comment:
<p>you should comment on whether the names of the categories matters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>